### PR TITLE
docs: add session history feature, folder explorer and folder create modal descriptions

### PR DESCRIPTION
**Pull request by ironAiken2**
### This PR is about updating `sessions` section for 24.09

**Changes:**
- Starting with Backend.AI version 24.09, NEO session launcher is used as the default session launcher. 
- Added session owner setting feature. 
- Added session histyro search function. 
- Added folder explorer and vfoler creation function in session launcher page.

**Related PRs:**
- https://github.com/lablup/backend.ai-webui/pull/2742 @yomybaby 
- https://github.com/lablup/backend.ai-webui/pull/2766 @yomybaby 
- https://github.com/lablup/backend.ai-webui/pull/2765 @yomybaby 
- https://github.com/lablup/backend.ai-webui/pull/2745 @ironAiken2 

- PDF[kr] : [연산 세션 — Backend.AI Web-UI User Guide 24.03 문서.pdf](https://github.com/user-attachments/files/17486329/Backend.AI.Web-UI.User.Guide.24.03.pdf)

- PDF[en] : [Compute Sessions — Backend.AI Web-UI User Guide 24.03 documentation.pdf](https://github.com/user-attachments/files/17486331/Compute.Sessions.Backend.AI.Web-UI.User.Guide.24.03.documentation.pdf)

### CODE REVIEW

**Comment by ironAiken2** [comment]
* **#86** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-docs-webui/86?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ironAiken2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by YEONFEEL96** [comment]
- Modified english expression on /sessions-section/sessions_all.rst
- Matched menu name with actual WebUI interface
https://github.com/lablup/backend.ai-docs-webui/pull/86/commits/36c6e987e9416b71f1528ff9f6efaf494dc15c3b

**Review by YEONFEEL96 — DISMISSED** [review]

File: docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Backend.AI Console User Guide 20.03\n"
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po:9]: Still reviewing the document, but shouldn't this to be 24.09? @ironAiken2 
```suggestion
"Project-Id-Version: Backend.AI Console User Guide 24.09\n"
```

**Review by YEONFEEL96 — CHANGES_REQUESTED** [review]
Question about version. Is it fine to leave this as 20.03?

File: docs/sessions_all/sessions_all.rst [hunk 2]
```diff
@@ -2,43 +2,40 @@
...
-refer to the following `link <https://webui.docs.backend.ai/en/23.09_a/sessions_all/sessions_all.html>`_.
+Click the 'START' button to start a new compute session.
 
 .. image:: launch_session_1.png
+   :width: 850
    :align: center
 
-In the first page, you need to choose the type of session, interactive or batch.
-And you can set the session name, which is optional.
+In ther first page, users can select the type of session, 'interactive' or 'batch'.
```
Comment (YEONFEEL96) [inline docs/sessions_all/sessions_all.rst:28]:
```suggestion
In the first page, users can select the type of session, 'interactive' or 'batch'. 
```

File: docs/sessions_all/sessions_all.rst [hunk 3]
```diff
@@ -51,136 +48,160 @@ And you can set the session name, which is optional.
...
-Now, we have reached the last page. You can view information of session(s) to create,
+On the last page, users could view information of session(s) to create,
 such as environment itself, allocated resources, mount information,
-environment variables set on the previous pages, preopen ports, etc.
-After confirming the settings, click the Launch button. If there's anything you'd like
-to change, you can return to the previous page by clicking the Previous button, or click
-the Edit button located at the top right of each card to go directly to the relevant page.
+environment variables set on the previous pages, preopen ports, etc.,
+Review the settings, users could launch the session by clicking 'Launch' button. 
+Click the 'Edit' button located at the top right of each card to redirect to relevent page.
```
Comment (YEONFEEL96) [inline docs/sessions_all/sessions_all.rst:201]:
```suggestion
Click the 'Edit' button located at the top right of each card to redirect to relevant page.
```

**Review by YEONFEEL96 — CHANGES_REQUESTED** [review]
Changed two small typos in english document.

**Review by YEONFEEL96 — APPROVED** [review]
LGTM!

**Pull request merged**